src/tools/dev/scripts/bv_support/bv_mfem.sh

```diff
@@ -20,18 +20,24 @@ function bv_mfem_depends_on
     if [[ "$DO_CONDUIT" == "yes" ]] ; then
         depends_on="$depends_on conduit"
     fi
+    if [[ "$DO_FMS" == "yes" ]] ; then
+        depends_on="$depends_on fms"
+    fi
 
     echo $depends_on
 }
 
 function bv_mfem_info
 {
-    export MFEM_VERSION=${MFEM_VERSION:-"4.0"}
+    # NOTE: we are using a special version of MFEM that has not yet been in an MFEM release.
+    export MFEM_VERSION=${MFEM_VERSION:-"Add_FMS_support"}
```

Comment:
Since this is a special version a ticket should be entered to update as soon as the new version with proper support is available.